tests/test_utils.py

```diff
@@ -36,11 +36,20 @@ class TestUtils:
     with open("tests/files/foo.gif", "rb") as fp:
         body_gif = fp.read()
 
-    def get_byte_seq(self, filename):
-        if not os.path.isfile(f"tests/files/{filename}"):
-            return bytes.fromhex(filename)
+    def get_byte_seq(self, seq):
+        if isinstance(seq, tuple):
+            if not os.path.isfile(f"tests/files/{seq[0]}"):
+                byte_seq = b"".join(
+                    value if isinstance(value, bytes) else bytes.fromhex(value) for value in seq
+                )
+            else:
+                byte_seq = seq[0]
+        elif not os.path.isfile(f"tests/files/{seq}"):
+            byte_seq = bytes.fromhex(seq)
         else:
-            return filename
+            byte_seq = seq
+
+        return byte_seq
```

Comment:
One possible further change: if it’s a file, read the file and return its content.

Also, if it’s a tuple, I think we can assume it is not a filename.

Reply:
Done!